### PR TITLE
remove styxkey, add modernizr

### PR DESCRIPTION
**Pull request by nataliejeremy**
This closes issue #556 @timani please review!

### CODE REVIEW

File: source/docs/articles/sites/varnish/caching-advancedtopics.md [hunk 1]
```diff
@@ -42,35 +42,20 @@ Drupal uses SESS-prefixed cookies for its own session tracking, so be sure to na
 ### WordPress Sites
 WordPress does not use PHP session cookies; however, some themes and plugins do. If you are using a theme or plugin that requires PHP sessions, you can install [Pantheon-sessions](https://wordpress.org/plugins/wp-native-php-sessions/ "Panthon Session WordPress plugin"). It is designed to handle the naming properly.
 
-## Geolocation, Referral Tracking, Content Customization, and Cache Segmentation Using STYXKEY
+## Geolocation, Referral Tracking, Content Customization, and Cache Segmentation
```
Comment (nataliejeremy) [inline source/docs/articles/sites/varnish/caching-advancedtopics.md]: @timani Does this header still work? If not please suggest a new one.

**Comment by davidstrauss** [comment]
-1 STYXKEY is actively used for capabilities that are unrelated to the replacement methods.

**Comment by timani** [comment]
It is perhaps the biggest source of pain for users implementing this. I have seen almost every case use HTML5 to handle this leave satisfied vs those who continue struggling with this.

We can keep it but we should make it clear mark it the less prefered option. #1 options should be the modern solution built in to most browsers

**Comment by nataliejeremy** [comment]
@davidstrauss @timani Please review and +1 or comment on the changes.

**Comment by nataliejeremy** [comment]
@davidstrauss Can you +1 if you approve the changes?

**Comment by davidstrauss** [comment]
> It is perhaps the biggest source of pain for users implementing this. I have seen almost every case use HTML5 to handle this leave satisfied vs those who continue struggling with this.

STYXKEY was not added for the purpose of fake responsive design; it is no surprise users don't have a good time trying to use it that way. It is around to facilitate page keying for the many other use cases for caching arbitrary page variants. +1 on restoring docs for it, though people should be warned that it's not for fake responsive design.

**Pull request merged**